From 1daca7b534482cdc3a6e0a95ab2ff3db57a47c32 Mon Sep 17 00:00:00 2001
From: Jeremy Huddleston <jeremyhu@apple.com>
Date: Mon, 25 Oct 2010 12:33:32 -0700
Subject: [PATCH] Move -I CFLAGS to CPPFLAGS

This will cause the build to use the included headers rather than installed
ones.

Signed-off-by: Jeremy Huddleston <jeremyhu@apple.com>
---
 src/Makefile.am |   10 +++++++++-
 1 files changed, 9 insertions(+), 1 deletions(-)

diff --git a/src/Makefile.am b/src/Makefile.am
index a77bb13..3bc4b29 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -56,7 +56,9 @@ COMMON_SOURCES = \
 	XawI18n.h
 
 COMMON_CFLAGS = \
-	$(CWARNFLAGS) \
+	$(CWARNFLAGS)
+
+COMMON_CPPFLAGS = \
 	-I${top_srcdir}/include \
 	-I${top_srcdir}/include/X11/Xaw \
 	-DPROJECT_ROOT=\"$(prefix)\"
@@ -65,6 +67,9 @@ if BUILD_XAW6
 
 lib_LTLIBRARIES += libXaw6.la
 
+libXaw6_la_CPPFLAGS =			\
+	$(COMMON_CPPFLAGS)
+
 libXaw6_la_CFLAGS =			\
 	$(COMMON_CFLAGS)		\
 	-DOLDXAW			\
@@ -100,6 +105,9 @@ if BUILD_XAW7
 
 lib_LTLIBRARIES += libXaw7.la
 
+libXaw7_la_CPPFLAGS =			\
+	$(COMMON_CPPFLAGS)
+
 libXaw7_la_CFLAGS =			\
 	$(COMMON_CFLAGS)		\
 	-DXAW7				\
-- 
1.7.3.1

